server/client_rest.go

```diff
@@ -59,9 +60,29 @@ func (client *clientRest) SessionCreate(nodeKey string) (session dto.Session, er
 	response, err := client.doRequest("POST", "client_create_session", dto.SessionStartRequest{
 		NodeKey: nodeKey,
 	})
+
 	if err == nil {
 		defer response.Body.Close()
+
 		err = parseResponseJson(response, &session)
+
+		if err != nil {
+			return
+		}
+
+		// this is a temporary solution for pulling the VPN config
```

Comment:
1. Why not to add to Session DTO?
2. I think endpoint `/client_create_session` is still unchanged. Why to change it?